src/core/ext/client_config/resolver_result.c

```diff
@@ -39,20 +37,20 @@
 
 struct grpc_resolver_result {
   gpr_refcount refs;
-  grpc_lb_policy *lb_policy;
+  grpc_lb_policy* lb_policy;
```

Comment:
I believe that clang just uses whichever form is most common in the file.  I personally prefer the space after the `*`, because I think it's part of the type, not part of the name, and since I was cleaning up this file anyway, I went ahead and made this change.